ames/block-template-parts/footer.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"20px";"bottom":"20px"}}}} -->
```

Comment:
This footer looks very similar to the one that ships in Blockbase. Why do we need a different one?

Reply:
Yeah, looks like it's not needed. I'll remove it.